src/main/java/org/humancellatlas/ingest/core/MetadataDocument.java

```diff
@@ -101,4 +121,12 @@ public MetadataDocument enactStateTransition(ValidationState targetState) {
 
         return this;
     }
+
+    @JsonIgnore
+    public SubmissionEnvelope getLatestSubmissionEnvelope(){
+        if(this.submissionEnvelopes.size() > 0){
+            return this.submissionEnvelopes.get(this.submissionEnvelopes.size() - 1);
+        }
+        return null;
+    }
```

Comment:
I think we should change this - I *think* you are always trying to get the open submission here, right?

If so, why not change this method to `getOpenSubmissionEnvelope()` and loop over the submission envelopes, looking for one that is still open (i.e. not yet submitted).  Bonus points: add an `isOpen()` check to `SubmissionEnvelope`.